src/UseFlash.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { Flash } from './Flash';
+
+export const UseFlash = () => {
+  return (
+    <div>
+      <Flash render={(value, valueFormatter) => <div>{valueFormatter(value)}</div>} />
```

Comment:
Currently use `<div>{valueFormatter(value)}</div>` instead of a component like 
`<Flash render={(value, valueFormatter) => (
       <Component {...this.props} value={value} valueFormatter={valueFormatter} />
)} />`